src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -568,6 +645,22 @@ private static void handleKeyValueConfig(TestInputConfiguration.Builder testInpu
         }
     }
 
+    public static Map<String, String> getDefaultProperties(String filePath, String moduleName)
```

Comment:
```suggestion
    public static Map<String, String> getDefaultProperties(String fullyQualifiedClassName)
```
thats all you need. No need to DTO inside the body. Give whats actually required.